dsref/ref.go

```diff
@@ -2,6 +2,9 @@ package dsref
 
 // Ref is a reference to a dataset
 type Ref struct {
+	// ID is the canonical identifer for a dataset history, in Qri canonical IDs
+	// are logbook InitIDs
+	ID string `json:"id,omitempty"`
```

Comment:
I think it's probably a good thing to add this to ref, but calling it just `ID` is ambiguous. If it's an initID, it should be called that.

Reply:
Sounds good! InitID it is.